lib/services/jira.rb

```diff
@@ -378,12 +383,16 @@ def update_issue(issue_info, resource, initiative, version, parent)
 
     if @feature == resource
       # Only update custom_fields and due dates for features, not requirements.
-      # This will cause an issue with two-way field syncing if we constantly overwrite requirements' custom fields
+      # This will cause an issue with two-way field syncing if we constantly
+      # overwrite requirements' custom fields
       issue.fields
         .merge!(mapped_custom_fields(@feature, issue_type))
```

Comment:
That is only for create, for update this is only run when the resource == @feature, not the same as the create code where the custom fields from the feature are added to a requirement.